shipyard-controller/models/expanded_project_old.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import apimodels "github.com/keptn/go-utils/pkg/api/models"
+
+type ExpandedProjectOld struct {
```

Comment:
please add a short explanation of why we have the `Old` struct in addition to the updated model, and what the differences are - otherwise, this might confuse us in a couple of months when we rediscover it :)